### DISCUSSION TITLE
Is it possible to host window view in flutter widget?

### DISCUSSION THREAD

**Discussion by tashi146** [halildurmus/win32#528] [General]
I am working on plugin which require native view. I have done the changes for android and ios with help of [platform integration](https://docs.flutter.dev/development/platform-integration/ios/platform-views). Now I am trying same with window and I can see there is no official support, There is also an open issue [here](https://github.com/flutter/flutter/issues/107518). So now my question is "Is there anyway to host native window view(such as native mapview,webview) using win32 package?".

**Comment by Far-Se** [comment]
https://github.com/alexmercerind/flutter_native_view

**Reply by tashi146** [reply]
I have used this before,I can see we can add window local applications such as vlc, device explorer etc on flutter with HWND. But mine is different, I have window project in c++, I want to host same native UI in flutter just like [platform integration](https://docs.flutter.dev/development/platform-integration/ios/platform-views) in Android and IOS
